DOC-5378 Add includes for Preview/Private Preview callouts (#15015)
cockroachdb/docs: _includes/feature-phases/private-preview.md

{{site.data.alerts.callout_info}}

> **This feature is in private preview**, and only available to select users. The interface and output are subject to change. We do not recommend using this feature in production.

Reviewer: Suggesting a rephrase, because I find the current version a bit confusing ("preview" implies that it will eventually become available, which doesn't line up with "not for production").

```suggestion
**This feature is in private preview**, and is only available to select users. The interface and output are subject to change. This feature is not intended for production use.
```

Reviewer 2: Nit: I don't think you need the comma before the first "and". 

Is "production use" how we usually talk about this? Would just "in production" or "for production workloads" be better?
